src/cargo/util/auth/mod.rs

```diff
@@ -500,7 +500,7 @@ fn credential_action(
     let name = if sid.is_crates_io() {
         Some(CRATES_IO_REGISTRY)
     } else {
-        sid.alt_registry_key()
+        sid.registry_key()
     };
```

Comment:
It looks like `registry_key` should always be `crates-io` when `is_crates_io()` is true.

Can this `if` be removed and replaced with `let name = sid.registry_key()`?

There are a few other places with this pattern too.